requirements.txt

```diff
@@ -3,7 +3,7 @@ matplotlib==2.1.1
 numpy==1.13.1
 numpydoc==0.7.0
 pandas==0.21.1
-pymc3==3.2
+pymc3==3.3
```

Comment:
Why not go to 3.4.1 directly?

Reply:
Ha, I just did pip freeze and that was what was installed at the time. I was mostly trying to get my slides ready.